backend/src/test/java/gov/cdc/usds/simplereport/service/TenantDataAccessServiceTest.java

```diff
@@ -0,0 +1,74 @@
+package gov.cdc.usds.simplereport.service;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+import static org.junit.jupiter.api.Assertions.assertTrue;
+
+import gov.cdc.usds.simplereport.config.AuthorizationProperties;
+import gov.cdc.usds.simplereport.config.authorization.OrganizationRole;
+import gov.cdc.usds.simplereport.config.authorization.OrganizationRoleClaims;
+import gov.cdc.usds.simplereport.db.model.ApiUser;
+import gov.cdc.usds.simplereport.db.model.Organization;
+import java.util.HashSet;
+import java.util.Optional;
+import java.util.Set;
+import org.apache.commons.collections.CollectionUtils;
+import org.junit.jupiter.api.Test;
+import org.springframework.beans.factory.annotation.Autowired;
+
+class TenantDataAccessServiceTest extends BaseServiceTest<TenantDataAccessService> {
+
+  @Autowired private ApiUserService _userService;
+  @Autowired private AuthorizationProperties _authProperties;
+
+  @Test
+  void addTenantDataAccess_success() {
+    ApiUser apiUser = _userService.getCurrentApiUserInContainedTransaction();
+    Organization org = _dataFactory.createValidOrg();
+    String justification = "Test justification";
+
+    Optional<OrganizationRoleClaims> claimsOpt =
+        _service.addTenantDataAccess(apiUser, org, justification);
+    assertTrue(claimsOpt.isPresent());
+
+    OrganizationRoleClaims claims = claimsOpt.get();
+    assertEquals(org.getExternalId(), claims.getOrganizationExternalId());
+    assertTrue(
```

Comment:
i was under the impression you could just do a normal assertEquals for set comparison